core/model/build.gradle.kts

```diff
@@ -19,6 +19,10 @@ plugins {
     id("kotlin")
 }
 
+kotlin {
+    jvmToolchain(11)
```

Comment:
Why was this line of code added in the `core/model` module?

Reply:
Because by default this kotlin module uses version 1.8 as you saw in the logs.